code/datums/brain_damage/paranoia.dm

```diff
@@ -0,0 +1,131 @@
+/datum/brain_trauma/special/renegade
+	name = "Paranoid Psychosis"
+	desc = "Patient has a psychotic disorder, becoming extremely paranoid of people around him."
+	scan_desc = "paranoid psychosis"
+	gain_text = "If you see this message, make a github issue report. The trauma initialized wrong."
+	lose_text = span_warning("You feel safe once again.")
+	can_gain = TRUE
+	random_gain = FALSE
+	resilience = TRAUMA_RESILIENCE_SURGERY
+	var/datum/antagonist/renegade/antagonist
+	///If someone is very close to us at the moment
+	var/being_viewed = FALSE
+	var/total_time_armed = 0 //just for round end fun
+	var/time_close = 0
+	///Did the person had bad touch before?
+	var/had_bad_touch = FALSE
+
+/datum/brain_trauma/special/renegade/on_gain()
+	gain_text = span_warning("Everyone around you seems suspicious, are they planning something against you? You cannont trust anyone, you must protect yourself by any means!")
```

Comment:
i don't see a need for gain text to be set here in all honesty

Reply:
> i don't see a need for gain text to be set here in all honesty

It was like that in the Obsessed, so maybe there is the reason. I just don't know them.